src/selectedtests/task_mappings/create_task_mappings.py

```diff
@@ -23,6 +25,49 @@
 TASK_BUILDS_KEY = "builds"
 
 
+def generate_task_mappings(
+    evg_api: EvergreenApi,
+    evergreen_project: str,
+    version_limit: VersionLimit,
+    source_file_regex: str,
+    module_name: str = None,
+    module_source_file_regex: str = None,
+    build_variant_regex: str = None,
+) -> Tuple[List[Dict], str]:
+    """
+    Generate task mappings for an evergreen project and its associated module if module is provided.
+
+    :param evg_api: An instance of the evg_api client.
+    :param evergreen_project: The name of the evergreen project to analyze.
+    :param version_limit: The point in time at which to start analyzing versions of the project.
```

Comment:
We have had long conversations about this name already. `starting_point` is the exact opposite of how I personally think about this (and the opposite of how the code works, we iterate until we hit this version so calling it starting anything seems confusing). The consensus seemed to be to avoid time based terminology, so we went with `limit`.

Reply:
The current docstring is: __The point in time at which to start analyzing versions of the project__. So that's very confusing.

Reply:
We should make the doc for that argument `How many versions back you want to go.` or such.

Reply:
I like `The point at which to start analyzing versions of the project`, I will change it to that

Reply:
I assumed the parameter was starting_point based on the current pydoc. 

If '_version_limit_' is the correct name and semantics then the pydoc should be more like 'The point in time to analyzing to.' (or something like that).